### ISSUE TITLE
NaNs generated after running RaPId

### ISSUE THREAD

**Issue by mpodlaski** [ALSETLab/RaPId#53]
I created a new simulink and script file for 64 bit version of the Mostar FMU. When I run the script, there's a bunch of NaNs output in the command window after the rapidObject.runIdentification() line (line 78) is run. This is in the examples/mostar/mostar.m script

**Comment by lvanfretti** [comment]
@janlav please take a look at this while you update the GUI.

**Comment by janlav** [comment]
@mpodlaski Hi, can you confirm that you are getting the correct values from the FMUmodel block to the Scopes when you run the Simulink model manually?

**Comment by mpodlaski** [comment]
@janlav Yes I am

**Comment by janlav** [comment]
@mpodlaski that is interesting, I assume the reference input is also behaving correct?
Does it say anything before or after about what these NaNs are, i.e. does it say anything about errors, iteration numbers,  or if the NaNs are fitness values or parameter estimates?

**Comment by lvanfretti** [comment]
I've got some problems running the example in my new setup.
First, we should modify the "dataMeasuredS", "From Workspace" block using instead an input defining [time signal] as per the attachment. This would require the user to load the "dataMeasuredO.mat" file to simulate without running RaPId.

I need to generate the FMU myself because my Dymola license is not being recognized, so 
[sim_model_64bit.mdl.zip](https://github.com/ALSETLab/RaPId/files/1782757/sim_model_64bit.mdl.zip)
I'll follow up next time we talk to make sure this example works well.

**Comment by mpodlaski** [comment]
@janlav everything is behaving correct. It doesn't tell us where the NaNs are coming from

**Comment by mpodlaski** [comment]
Update!! I figured out the issue is the Mostar dataset doesn't have a uniform sampling rate. The reference data needs to be updated. 

What essentially happens is in rapid_objectiveFunction.m on lines 40 and 41:
`        referenceResults = referenceResults(round(rapidSettings.experimentSettings.t_fitness_start/rapidSettings.experimentSettings.ts) +1:round(rapidSettings.experimentSettings.tf/rapidSettings.experimentSettings.ts),:);
        simulatedResults = simulatedResults(round(rapidSettings.experimentSettings.t_fitness_start/rapidSettings.experimentSettings.ts) +1:round(rapidSettings.experimentSettings.tf/rapidSettings.experimentSettings.ts),:);
`
The reference and simulated dataset aren't equal because of the sampling time. I noticed the sampling time ranges from 0.0044 - 0.0051 seconds in the reference file.
